src/librustc_resolve/lib.rs

```diff
@@ -1969,10 +1996,27 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                 Err(())
             }
         } else {
+
+            // find possible candidates
+            let trait_name = trait_path.segments.last().unwrap().identifier.name;
+            let candidates: Vec<_> =
+                self.lookup_suggestions(
+                    trait_name,
+                    LookupTarget::AllCrates,
+                    Namespace::TypeNS,
+                    move |def: Def|
+                        match def {
+                            Def::Trait(_) => true,
+                            _             => false
+                        }
```

Comment:
```rust
 self.lookup_suggestions(
     trait_name,
     LookupTarget::AllCrates,
     Namespace::TypeNS,
     |def| match def {
         Def::Trait(_) => true,
         _ => false,
     }, // < add a trailing comma here
+    new_argument, // so that we don't have add it while adding a new argument
 )
```